lib/Format/UnwrappedLineFormatter.cpp

```diff
@@ -853,7 +853,14 @@ UnwrappedLineFormatter::format(const SmallVectorImpl<AnnotatedLine *> &Lines,
          TheLine.First->TokenText == "qWarning") &&
          TheLine.Last && TheLine.Last->is(tok::semi))
       ShouldFormat = false;
-    // DVLT SPECIFIC
+
```

Comment:
so the old code with all token check on q and t logger category are useless now?